phpcs.xml

```diff
@@ -14,4 +16,24 @@
 	<rule ref="WordPress">
 		<exclude name="WordPress.VIP" />
 	</rule>
+
+	<rule ref="WordPress.WP.I18n">
+		<properties>
+			<!-- TGMPA itself uses `tgmpa`, the example file uses `theme-slug`. -->
+			<property name="text_domain" type="array" value="tgmpa,theme-slug" />
+		</properties>
+	</rule>
+
+	<rule ref="WordPress.WP.DeprecatedFunctions">
+		<properties>
+			<property name="minimum_supported_version" value="3.7" />
+		</properties>
+	</rule>
+
+	<rule ref="WordPress.WP.DeprecatedFunctions">
```

Comment:
Grr.. and I forgot that that last sniff hasn't been merged yet... oh well.. adding it to a backwing branch with other changes for when WPCS 0.12.0 comes out ;-)